src/verifiers/documentStatus/revocation.types.ts

```diff
@@ -20,3 +20,19 @@ export const RevocationStatus = Union(ValidRevocationStatus, InvalidRevocationSt
 export type RevocationStatus = Static<typeof RevocationStatus>;
 export const RevocationStatusArray = RunTypesArray(RevocationStatus);
 export type RevocationStatusArray = Static<typeof RevocationStatusArray>;
+
+export const OcspResponderRevocationStatus = Union(Literal("good"), Literal("revoked"), Literal("unknown"));
+export type OcspResponderRevocationStatus = Static<typeof OcspResponderRevocationStatus>;
+
+export enum OcspResponderRevocationReason {
```

Comment:
Where does all this come from ?

Reply:
for the runtype?

Reply:
the different codes

Reply:
https://datatracker.ietf.org/doc/html/rfc6960 page 6